apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -182,9 +83,10 @@ const ProjectCard = React.createClass({
 
   render() {
     const { projectData, currentGallery } = this.props;
-
-    const {type, channel, name} = this.props.projectData;
-    const url = currentGallery === 'personal' ? `/projects/${type}/${channel}/edit` : `/projects/${type}/${channel}`;
+    const { type, channel } = this.props.projectData;
```

Comment:
any reason you're not pulling name from here too?

Reply:
No strong reason.  I pulled type and channel from props because it seemed a little cleaner in the url, and then left everything else as `projectData.whatever` since there are quite a few attributes (name, studentName, updatedAt, etc). Is there a preference for all or nothing?

Reply:
oh I see you already pulled projectData from props. I think it's probably fine as is.